iroha/src/sumeragi.rs

```diff
@@ -338,4 +339,79 @@ mod tests {
         Sumeragi::sort_peers(&mut peers2, Some([1u8; 32]));
         assert_eq!(peers1, peers2);
     }
+
+    #[cfg(feature = "network-mock")]
+    #[async_std::test]
+    async fn all_peers_commit_block() {
+        let n_peers = 10;
+        let max_faults = 1;
+        let mut keys = Vec::new();
+        let mut ids = Vec::new();
+        let mut block_counters = Vec::new();
+        for i in 0..n_peers {
```

Comment:
Why can't we merge these two loops into one and remove vectors of keys and ids?

Reply:
It's because we have to pass all ids to both wsv and sumeragi. So we need to have them at first.

Reply:
Missed that - sry